core/templates/components/common-layout-directives/navigation-bars/side-navigation-bar.component.html

```diff
@@ -152,6 +160,9 @@
     letter-spacing: 0.3px;
     line-height: 16px;
   }
+  .oppia-sidebar-submenu-title {
+    font-size: 16px !important;
```

Comment:
Is there a way to do this without !important? I don't think we generally recommend using it.

Reply:
Done!